### DISCUSSION TITLE
How do I move down the file context?

### DISCUSSION THREAD

**Discussion by prince-of-maroc** [nvim-tree/nvim-tree.lua#2317] [Q&A] [answered]
When I open nvim-tree, at the top of the file explorer there is always an option to go up by one directory. However, I haven't found a way to move down one directory.

Sometimes I accidentally choose the option to go up by one directory, which makes the file explorer cluttered with a bunch of other
files that are not relevant to my project. Since I haven't found a way to lower the context back to the directory I want, the only solution I have found is to quit nvim and re-open it in the correct file context. In the interest of saving a few seconds, is there a way to move the file context down?

**Comment by gegoune** [comment, marked as answer]
Default mapping set by `config.mappings.default_on_attach()` for that is: `<C-]>`.
